src/components/Footer/index.js

```diff
@@ -4,7 +4,7 @@ export default function Footer() {
   return (
     <div className="bg-dark text-center">
       <div className="col-12" style={{padding: 10, color: 'white'}}>
-        <p>© 2017 - 2023 Pivotal Software, Inc. All Rights Reserved. See <a href="https://www.pivotal.io/terms-of-use">Terms of Use</a> and <a href="https://www.pivotal.io/privacy-policy">Privacy Policy</a>.</p>
+        <p>© 2017 - 2023 VMware, Inc. All Rights Reserved. See <a href="https://www.vmware.com/help/legal.html">Terms of Use</a> and <a href="https://www.pivotal.io/privacy-policy">Privacy Policy</a>.</p>
```

Comment:
Could you update the link of the privacy policy to https://www.vmware.com/help/privacy.html